api/src/main/java/gov/cms/ab2d/api/controller/BulkDataAccessAPI.java

```diff
@@ -112,6 +118,12 @@
         return returnStatusForJobCreation(job);
     }
 
+    private void checkIfInMaintenanceMode() {
+        if (propertiesService.isInMaintenanceMode()) {
+            throw new InMaintenanceModeException("The system is currently in maintenance mode. Please try the request again later.");
```

Comment:
Would the `ErrorHandler` automatically handle this new `InMaintenanceModeException` ?

Reply:
Yes since it extends RuntimeException, which is a subclass of Exception, and then matches the line `@ExceptionHandler(Exception.class)`